corehq/form_processor/interfaces/ledger_processor.py

```diff
@@ -17,10 +17,10 @@ def __new__(cls, to_save=None, to_delete=None):
         )
 
     def commit(self):
```

Comment:
Also this preserves the previous ordering since before this PR there would have been one `StockModelUpdateResult` for the delete followed by one for the save. Now there is just one.